src/sign/interactive_sign/participant.rs

```diff
@@ -488,6 +491,21 @@ mod tests {
             )
             .is_ok());
 
+        // Check we are able to create a recoverable signature.
```

Comment:
This code ensures we can successfully generate the recovery ID and derive the public key from that recovery ID on this unit test.

Reply:
Nope, just wanted to point it out!